### PR TITLE
docs: change rollappd to rollapp-wasm and update README

### LINKED ISSUE #52: Update readme
The readme is outdates.
We should copy the relevant parts from the evm rollapp template.

### PR DESCRIPTION
**Pull request by trinitys7**
## Description

Closes #52

### CODE REVIEW

**Review by anhductn2001 — APPROVED** [review]

File: README.with-advanced-features.md [hunk 1]
```diff
@@ -121,19 +144,13 @@ sh scripts/settlement/generate_denom_metadata.sh
...
 
-
 ```shell
 sh scripts/settlement/add_genesis_accounts.sh
 ```
 
 ### Register rollapp on settlement
 
 ```shell
-# for permissioned deployment setup, you must have access to an account whitelisted for rollapp
```
Comment (omritoptix) [inline README.with-advanced-features.md:132]: why is this part removed?

**Review by omritoptix — COMMENTED** [review]
2 things I notice:
1. don't see that we change to 3s block time before running the ibc script
2. rollapp-evm only has one readme. Ideally wasm should only have one readme as well.

**Review by omritoptix — APPROVED** [review]
